jobs/mongodb_migration/src/mongodb_migration/migrations/_20230216141000_queue_split_names_from_streaming.py

```diff
@@ -18,29 +18,55 @@
 class MigrationQueueUpdateSplitNames(Migration):
     def up(self) -> None:
         logging.info(
-            f"Rename unicity_id field from Job[{split_names}][<dataset>][<config>][<split>] to"
-            f" Job[{split_names_from_streaming}][<dataset>][<config>][<split>]"
+            f"Rename unicity_id field from Job[{split_names}][<dataset>][<config>][None] to"
+            f" Job[{split_names_from_streaming}][<dataset>][<config>][None] and change type from {split_names} to"
+            f" {split_names_from_streaming}"
         )
 
-        for job in Job.objects(type=split_names):
-            job.update(unicity_id=f"Job[{split_names_from_streaming}][{job.dataset}][{job.config}][{job.split}]")
-
-        logging.info(f"Rename type field from {split_names} to {split_names_from_streaming}")
         db = get_db("queue")
-        db["jobsBlue"].update_many({"type": split_names}, {"$set": {"type": split_names_from_streaming}})
+        db["jobsBlue"].update_many(
+            {"type": split_names},
+            [
+                {
+                    "$set": {
+                        "unicity_id": {
+                            "$replaceOne": {
+                                "input": "$unicity_id",
+                                "find": f"Job[{split_names}]",
```

Comment:
I think we want to change the prefix, right?

```suggestion
                                "find": f"Job[{split_names}",
```